python/lsst/sconsUtils/eupsForScons.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 try:
-    from eups import *
+    from eups import *  # noqa F403 F401 Importing everything into this namespace
```

Comment:
Now that I try it, I think the current situation is to be preferred. As soon as you import the EUPS versions, the linter complains about the redefinition of symbols later in the file and you have to `# noqa` those. The current approach has fewer noqa lines.